Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -238,6 +244,27 @@ + (void)setInAppPurchaseEventName:(NSString *)event
     LP_END_TRY
 }
 
++ (NSDictionary *) getDefaultAppKeysPlist
+{
+    NSString *plistFilePath = [[NSBundle mainBundle] pathForResource:kAppKeysFileName ofType:kAppKeysFileType];
+    if (plistFilePath == nil) {
```

Comment:
Throwing error shouldnt be needed since we have to try to load the keys if file exists, otherwise we can assume that client will set the keys manually